### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by yenhao**
I want to merge my OP model!

### CODE REVIEW

File: train_text-level_gnn.py [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+# coding: utf-8
+import sys
+
+import pandas as pd
+import numpy as np
+import os
+from collections import Counter, defaultdict
+from nltk.tokenize import TweetTokenizer
+
+tkzr = TweetTokenizer()
```
Comment (yenhao) [inline train_text-level_gnn.py]: should move later not here

Comment (yenhao) [reply train_text-level_gnn.py]: because I am lazy 👍

**Review by eunicebes — COMMENTED** [review]
把 training file 寫漂亮一點ㄅXD
不要用 jupyter 的格式

**Comment by yenhao** [comment]
> 把 training file 寫漂亮一點ㄅXD
> 不要用 jupyter 的格式

okay

**Pull request merged**